src/mumble/SearchDialog.h

```diff
@@ -47,6 +47,20 @@ struct SearchResult {
 		// A search result is only valid if it has start and end
 		return begin >= 0 && length > 0;
 	}
+
+#if _MSC_VER == 1900
+	SearchResult()
+	{}
+	
+	SearchResult(int32_t begin, int32_t length, SearchType type, QString fullText, QString channelHierarchy)
+	{
+		this->begin = begin;
+		this->length = length;
+		this->type = type;
+		this->fullText = fullText;
+		this->channelHierarchy = channelHierarchy;
+	}
+#endif
```

Comment:
If the compiler does not support the construction like that, it does not follow the necessary cpp standard (Cpp14). As such we do not support that compiler.
Adding hacks like this will only be a temporary workaround as we want and will use the newer cpp features wherever we see it fit and thus the compilation is likely to fail soon after again anyway.

For that reason I consider workarounds like this pointless and they actually clutter the code base and make it even harder to parse (and the code base is a total mess already).

Reply:
> Adding hacks like this will only be a temporary workaround as we want and will use the newer cpp features wherever we see it fit and thus the compilation is likely to fail soon after again anyway.

Absolutely right. In my case there are only a few things that need to be fixed, otherwise it doesn't make sense.